crates/bevy_picking/src/events.rs

```diff
@@ -377,12 +383,9 @@ pub fn send_click_and_drag_events(
     mut down_map: Local<
         HashMap<(PointerId, PointerButton), HashMap<Entity, (Pointer<Down>, Instant)>>,
     >,
-    // Output
+    // Outputs used for further processing
     mut drag_map: ResMut<DragMap>,
-    mut pointer_click: EventWriter<Pointer<Click>>,
```

Comment:
Why remove the `EventWriter`s here but not in `pointer_events`? Above it looks like you're both sending an event and triggering observers, but here only triggering observers

Reply:
That is done because the events on other places are further processed and then turned into triggers here.

I removed the eventwriters that was not used for further processing within the picking crate